src/server/app/notifications/models/notification.server.model.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+let mongoose = require('mongoose'),
+	extend = require('mongoose-schema-extend'),
```

Comment:
Can remove this require since we're not using the mongoose-schema-extend 3rd party package and using built-in mongoose discriminators.